### DISCUSSION TITLE
How do I add custom html into the global html head section ? [v3]

### DISCUSSION THREAD

**Discussion by michabbb** [filamentphp/filament#8371] [Help] [answered]
I was unable to find any solution to this. I would like to add custom HTML like this to the global HTML head section:

```
<link rel="apple-touch-icon" sizes="57x57" href="/favicons/apple-icon-57x57.png.pagespeed.ic.E0C7f1uj7e.webp">
<link rel="apple-touch-icon" sizes="60x60" href="/favicons/apple-icon-60x60.png.pagespeed.ic.LD_BX2v7K9.webp">
<link rel="apple-touch-icon" sizes="72x72" href="/favicons/apple-icon-72x72.png.pagespeed.ic.4sArUS023J.webp">
<link rel="apple-touch-icon" sizes="76x76" href="/favicons/apple-icon-76x76.png.pagespeed.ic.izm67NdxB4.webp">
```

I know there is a method to set the favicon icon, but for me, this is more a general question.
Is there a way to extend the global html->head for every page ?

thanks!

**Comment by michabbb** [comment, marked as answer]
okay, seems like i have found the secret to this 😏 

➡️  `AppServiceProvider.php`

```php
public function boot(): void
    {
        Filament::registerRenderHook(
            'panels::head.start',
            static fn (): string => \View::make('favicons')->render()
        );
    }
```

[source](https://filamentphp.com/docs/3.x/support/render-hooks#registering-render-hooks)

**Reply by leandrocfe** [reply]
Please do not tag Dan. If he has a solution, he will respond to your question
https://filamentphp.com/docs/3.x/panels/configuration#render-hooks
